plugins/secretstores/docker/docker.go

```diff
@@ -0,0 +1,79 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package docker
+
+import (
+	_ "embed"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/secretstores"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+type Docker struct {
+	ID   string `toml:"id"`
+	Path string `toml:"path"`
+}
+
+func (*Docker) SampleConfig() string {
+	return sampleConfig
+}
+
+// Init initializes all internals of the secret-store
+func (d *Docker) Init() error {
+	if d.ID == "" {
+		return errors.New("id missing")
+	}
+	if d.Path == "" {
+		return errors.New("path missing")
+	}
```

Comment:
What about setting `/run/secrets` as default?

Reply:
Done. Default path will be set to `/run/secrets` and no error will be raised if the `path` parameter is not mentioned explicitly.